### PR TITLE
Loose VTK XML version constraint

### PR DESCRIPTION
**Pull request by henry2004y**
This PR attempts to loose the current constraint on the VTK XML version requirement, based on the discussion in #43.

### CODE REVIEW

File: src/ReadVTK.jl [hunk 1]
```diff
@@ -28,7 +28,7 @@ Hold all relevant information about a VTK XML file that has been read in.
 - `filename`: original path to the VTK file that has been read in
 - `xml_file`: object that represents the XML file
 - `file_type`: currently only `"UnstructuredGrid"` or `"ImageData"` are supported
-- `version`: currently only v1.0 is supported
+- `version`: currently only v1.0 and beyond are supported
```
Comment (sloede) [inline src/ReadVTK.jl:31]:
```suggestion
- `version`: VTK XML file format version (v1.0 or later)
```
I think there is no useful spec information on this available publicly, so we can just be very neutral about this

File: src/ReadVTK.jl [hunk 2]
```diff
@@ -213,7 +213,7 @@ Hold all relevant information about a Parallel VTK XML file that has been read i
 - `filename`: original path to the PVTK file that has been read in
 - `xml_file`: xml info
 - `file_type`: currently only `"PRectilinearGrid"` or `"PImageData"` are supported
-- `version`: currently only v1.0 is supported
+- `version`: currently only v1.0 and beyond are supported
```
Comment (sloede) [inline src/ReadVTK.jl:216]:
```suggestion
- `version`: VTK XML file format version (v1.0 or later)
```
as above

File: src/ReadVTK.jl [hunk 3]
```diff
@@ -258,7 +258,7 @@ function PVTKFile(filename; dir = "")
   end
 
   # Ensure correct version
-  @assert version == v"1.0"
+  @assert version ≥ v"1.0"
```
Comment (sloede) [inline src/ReadVTK.jl:261]:
```suggestion
  @assert version >= v"1.0"
```
as above

File: src/ReadVTK.jl [hunk 4]
```diff
@@ -323,7 +323,7 @@ function PVDFile(filename)
...
```
Comment (sloede) [inline src/ReadVTK.jl:326]:
```suggestion
  @assert version >= v"1.0"
```
as above

File: src/ReadVTK.jl [hunk 5]
```diff
@@ -125,7 +125,7 @@ function VTKFile(filename)
...
```
Comment (sloede) [inline src/ReadVTK.jl:128]:
```suggestion
  @assert version >= v"1.0"
```
We prefer to only use ASCII characters for code

**Review by sloede — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! With some minor changes, this LGTM.

File: src/ReadVTK.jl (hunk 1 above)
Comment (henry2004y) [reply src/ReadVTK.jl]: Yep

**Review by sloede — APPROVED** [review]

**Comment by sloede** [comment]
v0.1.9 has just been released with your fix included.

**Pull request merged**